.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Improvement
+about: Suggest an improvement to signac-docs
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!-- Please replace the text in the individual sections below. -->
+
+### Issue Description
+
+A concise description of the proposed improvement or the problem it solves.
+For example: "I would like to ..." or "It would be nice if it was easier to ...".
+
+### Proposed solution
```

Comment:
I'm wondering if we should speak strictly of "changes" rather than "solutions" in the context of documentation.